.evergreen/config.yml

```diff
@@ -469,9 +484,13 @@ tasks:
     - variant: macos
       name: build-and-test-and-upload
   commands:
+    - func: "fetch source"
     - command: shell.exec
       params:
-        script: mkdir all
+        script: |-
+          # this directory was created in "fetch source", but it isn't needed here
+          rm -rf libmongocrypt
```

Comment:
Is removal of this directory necessary? Otherwise I do not see the harm in leaving it as-is.

Reply:
I don't think that it's necessary, per se.  However, since the task has no other reason to have the source tree, it seemed like a good thing to keep everything tidy.

Reply:
OK.  I see what you are saying.  I don't think that a race condition like you are describing could occur based on how Evergreen works.  However, I don't have a strong conviction that the directory needs to be removed, so I'll drop the removal.